README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/expectedbehavior/acts_as_archival.svg?branch=master)](https://travis-ci.org/expectedbehavior/acts_as_archival)
```

Comment:
Build status is not more important than what the gem actually does. Move down below at least the introductory paragraphs? If you are thinking there will be more than one there can be a new `##` section.